server/api/utils.go

```diff
@@ -37,7 +37,7 @@ var MediaContentTypes = [...]string{
 
 // WriteFileResponse is copied from Mattermost server https://github.com/mattermost/mattermost/blob/f0121d4f23e065a05e63fd452486570271c6a28f/server/platform/shared/web/files.go#L38
 func WriteFileResponse(filename string, contentType string, contentSize int64, lastModification time.Time, webserverMode string, fileReader io.ReadSeeker, forceDownload bool, w http.ResponseWriter, r *http.Request) {
-	w.Header().Set("Cache-Control", "private, max-age=86400")
+	w.Header().Set("Cache-Control", "no-cache, no-store, must-revalidate")
```

Comment:
For my own knowledge, do you mind explaining this change?

Reply:
Previously, I was allowing clients to cache the file. As we can export a report from an in-progress survey, lets say I generated a report when there were 10 responses. Now there are 15 responses. If I generate the report again, the browser will return the cached file instead of fetching a fresh file. So, I disabled caching for files to force clients to always re-generate a report to avoid stale data.